### PR TITLE
Capitalize constants

### PR DESCRIPTION
**Pull request by MrKingMichael**
This a draft PR to track the codebase's refactoring. The goal is to follow Python's PEP 8 coding style guide and produce more readable and consistent code.

### CODE REVIEW

**Comment by MaxwellJung** [comment]
You also have to replace all the old constans with new constants. I think you're approach is very error prone because you're overwriting all the old constant names without actually replacing any of them.

**Comment by MaxwellJung** [comment]
You need to replace the constant names first before you start reordering them

File: src/constants/ow1_constants.opy [hunk 1]
```diff
@@ -2,176 +2,174 @@
...
+#!define OW1_MEI_ULT_COST 1610
+#!define OW1_PHARAH_ULT_COST 2100
+#!define OW1_REAPER_ULT_COST 2100
+#!define OW1_SOLDIER_ULT_COST 2310
+#!define OW1_SOMBRA_ULT_COST 1400
+#!define OW1_SYMMETRA_ULT_COST 1680
+#!define OW1_TORBJORN_ULT_COST 2142
+#!define OW1_TRACER_ULT_COST 1260
+#!define OW1_WIDOWMAKER_ULT_COST 1540
+#!define OW1_WINSTON_ULT_COST 1540
```
Comment (MaxwellJung) [inline src/constants/ow1_constants.opy:30]: This line should go with the tanks

**Comment by MaxwellJung** [comment]
I figured out a regex replace to do this easily

**Comment by MrKingMichael** [comment]
I'm in the process of uploading the updates files with all of the constants updated. Please refrain from merging your changes.

**Comment by MaxwellJung** [comment]
I updated all the constants to the format `OW[1-2](_[A-Z]+)+`

File: src/base_settings.opy [hunk 2]
```diff
@@ -55,105 +55,105 @@ settings {
     "heroes": {
         "allTeams": {
             "ana": {
-                "ultGen%": percent(ow2_ult_cost_ANA/ow1_ult_cost_ANA)
+                "ultGen%": percent(OW2_DVA_ULT_COST/OW1_ANA_ULT_COST)
```
Comment (MaxwellJung) [inline src/base_settings.opy:58]: this one's wrong

Comment (MrKingMichael) [reply src/base_settings.opy:58]: Good catch, sorry about that. Will fix.

File: src/ow1/passive/tank.opy [hunk 3]
```diff
@@ -9,36 +9,36 @@ playervar ult_percent_compensated
...
+    ult_cost[heroID(Hero.PHARAH)] = 1/(roundedPercent(OW2_PHARAH_ULT_COST/OW1_PHARAH_ULT_COST)/100)*OW2_PHARAH_ULT_COST
+    ult_cost[heroID(Hero.REAPER)] = 1/(roundedPercent(OW2_REAPER_ULT_COST/OW1_REAPER_ULT_COST)/100)*OW2_REAPER_ULT_COST
+    ult_cost[heroID(Hero.SOLDIER)] = 1/(roundedPercent(OW2_SOLDIER_ULT_COST/OW1_SOLDIER_ULT_COST)/100)*OW2_SOLDIER_ULT_COST
+    ult_cost[heroID(Hero.SOMBRA)] = 1/(roundedPercent(OW2_SOMBRA_ULT_COST/OW1_SOMBRA_ULT_COST)/100)*OW2_SOMBRA_ULT_COST
+    ult_cost[heroID(Hero.SYMMETRA)] = 1/(roundedPercent(OW2_SYMMETRA_ULT_COST/OW1_SYMMETRA_ULT_COST)/100)*OW2_SYMMETRA_ULT_COST
+    ult_cost[heroID(Hero.TORBJORN)] = 1/(roundedPercent(OW2_TORBJORN_ULT_COST/OW1_TORBJORN_ULT_COST)/100)*OW2_TORBJORN_ULT_COST
+    ult_cost[heroID(Hero.TRACER)] = 1/(roundedPercent(OW2_TRACER_ULT_COST/OW1_TRACER_ULT_COST)/100)*OW2_TRACER_ULT_COST
+    ult_cost[heroID(Hero.WIDOWMAKER)] = 1/(roundedPercent(OW2_WIDOWMAKER_ULT_COST/OW1_WIDOWMAKER_ULT_COST)/100)*OW2_WIDOWMAKER_ULT_COST
+    ult_cost[heroID(Hero.ANA)] = 1/(roundedPercent(OW2_DVA_ULT_COST/OW1_ANA_ULT_COST)/100)*OW2_DVA_ULT_COST
```
Comment (MaxwellJung) [inline src/ow1/passive/tank.opy:35]: same error here

**Comment by MaxwellJung** [comment]
I did a scan and it looks fine. Gonna have to see if it compiles though

**Comment by MaxwellJung** [comment]
I think we use this PR just for capitalizing constants, then submit another PR for other types of refactoring.

**Review by MaxwellJung — APPROVED** [review]

**Comment by MaxwellJung** [comment]
I confirmed that the compiled output is identical.

**Pull request merged**